setup.py

```diff
@@ -0,0 +1,9 @@
+from setuptools import setup, find_packages
+
+__version__ = (0, 3, 1)
+
+setup(
+    name = 'underScore',
+    packages = find_packages(),
+    # package_data = {'data': ['heroes.json', 'raw_data.json']}
+)
```

Comment:
So I know we had a conversation about a `setup.py` file but it looks like I was wrong.  We don't need this file here so we can safely get rid of it.